presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestRoles.java

```diff
@@ -561,6 +561,21 @@ public void testGrantAllRevokeSelect()
     }
 
     @Test(groups = {HIVE_CONNECTOR, ROLES, AUTHORIZATION, PROFILE_SPECIFIC_TESTS})
+    public void testGrantSelectRevokeAll()
+            throws Exception
+    {
+        final String tableName = "default.test_table";
+        onPresto().executeQuery("CREATE ROLE role1");
+        onPresto().executeQuery("GRANT role1 TO alice");
+        onPresto().executeQuery(format("CREATE TABLE %s (foo BIGINT)", tableName));
+        onPresto().executeQuery(format("GRANT SELECT ON %s TO role1", tableName));
```

Comment:
That fixed it.  Now I see where the documentation indicates that USER is the default if neither USER or ROLE is specified, but I think this should be emphasized more clearly in the docs.  I'm going to file a Jira ticket for that as I think we'll get support requests related to this otherwise.